src/sbt-test/skip-windows/caching/build.sbt

```diff
@@ -7,7 +7,7 @@ inThisBuild(
     scalacOptions += "-Yrangepos",
     scalacOptions += "-Ywarn-unused", // for RemoveUnused
     scalafixCaching := true,
-    scalafixDependencies += "com.geirsson" %% "example-scalafix-rule" % "1.2.0"
+    scalafixDependencies += "com.nequissimus" %% "sort-imports" % "0.5.3"
```

Comment:
I used another artifact as I wanted to have 2 different revisions published under the same organization/name